client/src/components/Dashboard/League/Stepper.jsx

```diff
@@ -40,6 +42,48 @@ const Stepper = () => {
             alignItems="center"
             justifyContent="center"
         >
+            {isDesktop ? (
+                <IconButton
+                    icon={<ArrowBackIcon />}
+                    size="md"
+                    boxSize={6}
+                    alignSelf="center"
+                    justifySelf="center"
+                    position="absolute"
+                    left="0rem"
+                    transform="translateX(-40%)"
+                    color="white"
+                    bg="greyBg"
+                    onClick={() =>
```

Comment:
might be worth abstracting this into a function since you use it twice here, again not a big deal though